test/quicapitest.c

```diff
@@ -96,9 +96,10 @@ static int test_quic_write_read(int idx)
                 goto end;
             if (idx >= 1) {
                 do {
+#if !defined(OPENSSL_NO_POSIX_IO) && !defined(OPENSSL_NO_SOCK)
                     if (!TEST_true(wait_until_sock_readable(ssock)))
                         goto end;
-
```

Comment:
Not sure how this is supposed to work?

All of this code depends on sockets, so it doesn't make sense to try and use it without sockets support.

Reply:
Instead I suggest you modify `qtest_support_blocking()` that is called above to return 0 in the case of `OPENSSL_NO_SOCK` (it already checks `OPENSSL_NO_POSIX_IO`). We should have an empty implementation of `wait_until_sock_readable()` that just returns 0 if not socket support is available.

Reply:
The problem is that we do not have the empty implementation so the build fails. Yes, an alternative fix is to add the empty implementation. I'll do that.

Reply:
BTW no-sock is not a problem - in that case we disable quic support completely.